code/game/machinery/shieldgen.dm

```diff
@@ -419,8 +419,8 @@
 		needs_power = TRUE
 		setDir(get_dir(gen_primary, gen_secondary))
 	for(var/mob/living/L in get_turf(src))
-		visible_message("<span class='danger'>\The [src] is suddenly occupying the same space as \the [L]!</span>")
-		L.gib()
+		visible_message("<span class='danger'>\The [src] zaps through \The [L]!</span>")
```

Comment:
Why is the now capitalized (and does that even matter in byond, I assume it does).
Also [L] will automatically insert the anyway.